cassandra-diagnostics-reporter-influx/src/main/java/io/smartcat/cassandra/diagnostics/reporter/InfluxReporter.java

```diff
@@ -106,10 +106,12 @@ public void report(Query queryReport) {
         logger.debug("Sending Query: {}", queryReport.toString());
         try {
             influx.write(dbName, retentionPolicy,
-                    Point.measurement(measurementName).time(queryReport.startTimeInMilliseconds, TimeUnit.MILLISECONDS)
+                    Point.measurement(measurementName)
+                            .time(queryReport.getStartTimeInMilliseconds(), TimeUnit.MILLISECONDS)
                             .tag("host", hostname).tag("id", UUID.randomUUID().toString())
-                            .addField("client", queryReport.clientAddress).addField("statement", queryReport.statement)
-                            .addField("value", queryReport.executionTimeInMilliseconds).build());
+                            .addField("client", queryReport.getClientAddress())
+                            .addField("statement", queryReport.getStatement())
```

Comment:
We need to differentiate update and select for this use case, so we need statement type at least to be tag (currently we are using class name). Probably statement will be for audit purpose and query type or something is "update", "Select", "delete" and we need this to be tag.

Current influx schema and Grafana presentation relies on statement to be tag, that is reason why I brought this up to have same behavior.